in.go

```diff
@@ -73,6 +73,16 @@ func Get(request GetRequest, github Github, git Git, outputDir string) (*GetResp
 	metadata.Add("message", pull.Tip.Message)
 	metadata.Add("author", pull.Tip.Author.User.Login)
 
+	labels := ""
+	for i := 0; i < len(pull.Labels); i++ {
+		labels += pull.Labels[i].Name
+
+		if i < len(pull.Labels)-1 {
+			labels += ", "
```

Comment:
You should use `strings.Join` here instead of rolling your own. 

But, I'm wondering if we should:
1. Not emit the metadata to be displayed by Concourse since it can be an arbitrary number. Right now you have limited it to 10 labels, but if Github allows it to be more, then I think it's more important to make sure the filter functionality is consistent (by fetching up to 100 labels) rather than being able to display it in the UI.
2. Fetch it and only write it to a file in `.git/resource`.
3. Not fetch the labels on `get` at all, since it's not being used by the resource outside of `check`.

I'm leaning towards number 2 or 3 here.

Reply:
> You should use strings.Join here instead of rolling your own.

:) Yeah went too old school. 

As for the other points, the only reason why I added the labels in the metadata is because I thought it might be useful for some users to see all of the labels being applied on the PR like so:

![image](https://user-images.githubusercontent.com/28300158/61381541-78bc6f80-a8ab-11e9-82e9-e10a40896bc7.png)

But I totally agree with you that the actual label filtering logic in the `check` is the most important part. We already know exactly what labels we will expect.

So I'm totally fine with not fetching the labels at all on `get`.